lib/vet360/contact_information/person_response.rb

```diff
@@ -9,10 +9,13 @@ class PersonResponse < Vet360::Response
 
       attr_reader :response_body
 
-      def initialize(status, response = nil)
-        @response_body = response&.body
+      def self.from(raw_response = nil)
+        @response_body = raw_response&.body
 
-        super(status, person: Vet360::Models::Person.build_from(@response_body&.dig('bio')))
+        new(
+          raw_response&.status,
+          person: Vet360::Models::Person.build_from(@response_body&.dig('bio'))
+        )
```

Comment:
This method is mostly just a refactor to reduce the amount of data that needs to be passed in, in addition to making it easier to instantiate instances of the response classes in error/test scenarios.